server/package.json

```
@@ -7,6 +7,7 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "node app.js",
     "dev": "node --watch app.js",
+    "seed": "node .\\prisma\\seed.js",
```

```suggestion
    "seed": "node ./prisma/seed.js",
```

linux generally uses forward slashes and windows can understand them too. I don't think we should have backslashes here